destinations/destinations.go

```diff
@@ -0,0 +1,34 @@
+package destinations
+
+import (
+	"io"
+	"net"
+	"os"
+)
+
+var (
+	connection io.WriteCloser
+)
+
+func NewRemote() (io.WriteCloser, error) {
+	// read remote logging destination from environment
```

Comment:
Maybe move the lines 15-18 into a function like `readLoggingDestination` and spare the comment.

Reply:
done